### PR TITLE
Add event for process exited unexpectedly

### PR DESCRIPTION
**Pull request by zbud-msft**
### Description of PR

Summary:
Fixes # (issue)26636373

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [ ] 202205

#### What is the motivation for this PR?

Add event for process-exited-unexpectedly. Killed critical process in snmp or acms container if internal image.

#### How did you do it?

Code change

#### How did you verify/test it?

Manual test/pipeline

### CODE REVIEW

File: tests/telemetry/events/host_events.py [hunk 1]
```diff
@@ -89,3 +108,26 @@ def mask_container(duthost):
...
+    duthost.shell("docker exec {} kill -9 {}".format(container, pid), module_ignore_errors=True)
+
+    # Wait until specified container is not running because of critical process exit
+    wait_until(30, 5, 0, is_container_down, duthost, container)
+
+    if change_autorestart:
+        duthost.shell("config feature autorestart {} disabled".format(container))
+
+    duthost.shell("systemctl reset-failed {}".format(container))
+    duthost.shell("systemctl restart {}".format(container))
```
Comment (liuh-80) [inline tests/telemetry/events/host_events.py:133]: If container not fully restart, will next test failed? suggest wait fully restart here.

**Review by liuh-80 — APPROVED** [review]

**Review by qiluo-msft — APPROVED** [review]

**Comment by bingwang-ms** [comment]
Hi @zbud-msft , can you fix the issue ID in the PR description ?
`26636373` is not a valid issue id.

**Comment by zbud-msft** [comment]
Hi @StormLiangMS, can we have this in 202311?

**Pull request merged**